app/stores/NetworkStatusStore.js

```diff
@@ -185,11 +185,13 @@ export default class NetworkStatusStore extends Store {
   };
 
   @action _updateLocalTimeDifference = async () => {
-    try {
-      const response = await this.localTimeDifferenceRequest.execute().promise;
-      runInAction('update time difference', () => (this.localTimeDifference = response));
-    } catch (error) {
-      runInAction('update time difference', () => (this.localTimeDifference = this.ALLOWED_TIME_DIFFERENCE + 1));
+    if (this.isConnected) {
+      try {
+        const response = await this.localTimeDifferenceRequest.execute().promise;
+        runInAction('update time difference', () => (this.localTimeDifference = response));
+      } catch (error) {
+        runInAction('update time difference', () => (this.localTimeDifference = this.ALLOWED_TIME_DIFFERENCE + 1));
```

Comment:
Does this mean that we show the "out of time sync" error screen anytime the request fails?

Reply:
Yes this is true. That is what we agreed in the beginning.
Do you think we should change this? 

@darko-mijic cc

Reply:
If we agreed to this behavior all is good 😉 just wanted to ask to be sure

Reply:
I am open to any suggestions - do you think we should change this logic?

Reply:
Hmm … im not sure, i guess this would only happen if the node is down? In that case we should be falling back to the "connecting" stage anyway, or am i mistaken there?

Reply:
@DominikGuzei you are right - in case we get disconnected user will be redirected to connecting screen. Still after discussion with @darko-mijic we decided to reset timeDifference to zero in case of error response from localTimeDifferenceRequest.
@tomothespian please fix this ☝️

Reply:
done!